File: my-folder/0004-median-of-two-sorted-arrays/solution.py
class Solution:
    def findMedianSortedArrays(self, nums1: List[int], nums2: List[int]) -> float:
        for i in nums2:
            nums1.append(i)
        nums1.sort()
        n=len(nums1)
        if len(nums1)%2==0:
            a=n//2
            return((nums1[a-1]+nums1[a])/2)
        elif len(nums1)%2!=0:
            a=n//2
            return nums1[a]

File: my-folder/0004-median-of-two-sorted-arrays/solution.cpp
class Solution {
public:
    double findMedianSortedArrays(vector<int>& nums1, vector<int>& nums2) {
        vector<int>v;
        merge(nums1.begin(),nums1.end(),nums2.begin(),nums2.end(),back_inserter(v));
        int n=v.size();
        double a;
        if(n%2==0){
            a=((v[n/2-1]+v[n/2])/2.0);
        }
        else{
            a=v[n/2];
        }
        return a;
    }
};
